streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java

```diff
@@ -379,6 +374,31 @@ public void shouldReleaseLockIfExceptionWhenLoadingCheckpoints() throws Exceptio
         }
     }
 
+    @Test(expected = StreamsException.class)
+    public void shouldThrowStreamsExceptionIfFailedToReadCheckpoints() throws Exception {
```

Comment:
This is the same as `shouldThrowStreamsExceptionIfFailedToReadCheckpointedOffsets`?